xbmc/guilib/GUIControl.cpp

```diff
@@ -177,6 +177,10 @@ void CGUIControl::Process(unsigned int currentTime, CDirtyRegionList &dirtyregio
 // 3. reset the animation transform
 void CGUIControl::DoRender()
 {
+  if (IsControlRenderable() &&
+      !m_renderRegion.Intersects(CServiceBroker::GetWinSystem()->GetGfxContext().GetScissors()))
+    return;
```

Comment:
it looks like the root cause for issue https://github.com/xbmc/xbmc/issues/25279 is lying here.
i can workaround it by not calling `m_renderRegion.Intersects(GetScissors())` for `GUICONTROL_TOGGLEBUTTON`

```diff
diff --git a/xbmc/guilib/GUIControl.cpp b/xbmc/guilib/GUIControl.cpp
index 7ae9728120..68cd9f48a3 100644
--- a/xbmc/guilib/GUIControl.cpp
+++ b/xbmc/guilib/GUIControl.cpp
@@ -180,7 +180,7 @@ void CGUIControl::Process(unsigned int currentTime, CDirtyRegionList &dirtyregio
 // 3. reset the animation transform
 void CGUIControl::DoRender()
 {
-  if (IsControlRenderable() &&
+  if (IsControlRenderable() && ControlType != GUICONTROL_TOGGLEBUTTON &&
       !m_renderRegion.Intersects(CServiceBroker::GetWinSystem()->GetGfxContext().GetScissors()))
     return;
```

but no idea how to fix it properly